src/main/kotlin/graphics/scenery/primitives/Atmosphere.kt

```diff
@@ -17,51 +17,66 @@ import kotlin.math.*
  * Implementation of a Nishita sky shader, applied to an [Icosphere] that wraps around the scene as a skybox.
  * The shader code is ported from Rye Terrells [repository](https://github.com/wwwtyro/glsl-atmosphere).
  * To move the sun with arrow keybinds, attach the behaviours using the [attachRotateBehaviours] function.
- * @param initSunDir [Vector3f] of the sun position. Defaults to sun elevation of the current local time.
- * @param emissionStrength Emission strength of the atmosphere shader. Defaults to 0.3f.
+ * @param initSunDirection [Vector3f] of the sun position. Defaults to sun elevation of the current local time.
+ * @param emissionStrength Emission strength of the atmosphere shader. Defaults to 1f.
  * @param latitude Latitude of the user; needed to calculate the local sun position. Defaults to 50.0, which is central Germany.
  */
-open class Atmosphere(initSunDir: Vector3f? = null, emissionStrength: Float = 0.3f, var latitude: Double = 50.0) :
+open class Atmosphere(
+    initSunDirection: Vector3f? = null,
+    var emissionStrength: Float = 1f,
```

Comment:
Write as 1.0f please, for consistency

Reply:
emissionStrength should also be a regular member variable, with a custom setter, so you can update the material properties automatically without the need for the `updateEmissionStrength()` method